### PR TITLE
manual: insert section to illustrate slot tinkering.

### PR DESCRIPTION
**Pull request by pdelfino**

### CODE REVIEW

File: source/manual.lisp [hunk 1]
```diff
@@ -46,6 +46,25 @@ class and slot documentation for the individual details.")
...
+    (:ul
+     (:li "Execute command " (command-markup 'describe-slot))
+     (:li "Type " (:code 'hints-alphabet))
+     (:li "Select " (:code "hints-alphabet") " (" (:code "user-web-mode") " class option).")
+     (:li "Press the button " (:code "Configure"))
+     (:li "Insert the string \"asfdghjkl\""))
+    (:p "This will make link-hinting more comfortable for this user. As
+non-exhaustive examples, a similar approach of customization can be applied to
+other slots such as " (:code "spell-check-language") ", which will expand the
+spelling-check feature to other languages besides English. Or changing slot "
```
Comment (jmercouris) [inline source/manual.lisp:65]: I believe two examples should suffice :-)

Comment (pdelfino) [reply source/manual.lisp]: Done!

**Comment by pdelfino** [comment]
I think it is good to go!

Besides removing one of the examples, I made the HTML list an ordered one, instead of unordered.

**Comment by jmercouris** [comment]
Thank you!

**Pull request merged**